cts/Makefile.am

```diff
@@ -56,7 +56,8 @@ cts_SCRIPTS		= CTSlab.py		\
 			  fence_dummy		\
 			  LSBDummy		\
 			  lxc_autogen.sh	\
-			  pacemaker-cts-dummyd
+			  pacemaker-cts-dummyd	\
+			  cts.conf
```

Comment:
cts.conf would go in dist_cts_DATA -- DATA b/c it's not executable, and dist b/c it's static rather than generated from a .in file (but beware, the only file currently in dist_cts_DATA is conditioned on upstart). BTW this should fix the CI issue.

Reply:
Done.